Swift/Document.swift

```diff
@@ -42,7 +42,7 @@ public class Document : DictionaryProtocol, Equatable, Hashable, Sequence {
     }
     
     /// The collection that the document belongs to.
```

Comment:
I don't know swift.  Does this create a publicly mutable property?  Is that what internal(set) prevents?

Reply:
Yes, that is what `internal(set) prevents` prevents it to be public.